lighthouse-core/report/html/renderer/details-renderer.js

```diff
@@ -61,9 +61,6 @@ class DetailsRenderer {
       case 'screenshot':
       case 'debugdata':
         return null;
-      // Fallback for old LHRs, where no type meant don't render.
-      case undefined:
-        return null;
```

Comment:
moved into `util.prepareReportResult()` so all the backwards compat stuff is in one place